test/module/irohad/synchronizer/synchronizer_test.cpp

```diff
@@ -104,11 +91,11 @@ TEST_F(SynchronizerTest, ValidWhenSingleCommitSynchronized) {
 
   auto wrapper =
       make_test_subscriber<CallExact>(synchronizer->on_commit_chain(), 1);
-  wrapper.subscribe([&test_block](auto commit) {
+  wrapper.subscribe([test_block](auto commit) {
```

Comment:
is it possible to pass by const ref?

Reply:
It's not so important for shared pointer.

Reply:
It is about ownership. Is the code require independent dependency or it used it for some limited time.